seqr/management/tests/reload_clinvar_all_variants_tests.py

```diff
@@ -0,0 +1,310 @@
+                        <MedGen CUI="C5882668" Name="CHEK2-related cancer predisposition"/>
+                    </TraitMapping>
+                    <TraitMapping ClinicalAssertionID="4961845" TraitType="Disease" MappingType="XRef" MappingValue="C0027672" MappingRef="MedGen">
+                        <MedGen CUI="C0027672" Name="Hereditary cancer-predisposing syndrome"/>
+                    </TraitMapping>
+                </TraitMappingList>
+            </ClinicalAssertionList>
+        </ClassifiedRecord>
+    </VariationArchive>
+    <VariationArchive VariationID="5603" VariationName="NM_007194.4(CHEK2):c.1283C>T (p.Ser428Phe)" VariationType="single nucleotide variant" Accession="VCV000005603" Version="104" RecordType="classified" NumberOfSubmissions="38" NumberOfSubmitters="36" DateLastUpdated="2025-06-29" DateCreated="2016-03-20" MostRecentSubmission="2025-06-29">
+        <ClassifiedRecord>
+            <SimpleAllele AlleleID="20642" VariationID="5603">
+                <Location>
+                    <SequenceLocation Assembly="GRCh38" Chr="MT" Accession="NC_000022.11" start="123" stop="123" display_start="123" display_stop="28695219" variantLength="1" positionVCF="123" referenceAlleleVCF="G" alternateAlleleVCF="A"/>
+                </Location>
+            </SimpleAllele>
+            <Classifications>
+                <GermlineClassification DateLastEvaluated="2025-04-01" NumberOfSubmissions="38" NumberOfSubmitters="36" DateCreated="2016-03-20" MostRecentSubmission="2025-06-29">
+                    <Description>Pathogenic/Likely pathogenic/Pathogenic, low penetrance/Established risk allele</Description>
+                </GermlineClassification>
+            </Classifications>
+        </ClassifiedRecord>
+    </VariationArchive>
+</ClinVarVariationRelease>
+'''
+
+
+@mock.patch('seqr.management.commands.reload_clinvar_all_variants.safe_post_to_slack')
+@mock.patch('seqr.management.commands.reload_clinvar_all_variants.logger.info')
+class ReloadClinvarAllVariantsTest(TestCase):
+    databases = '__all__'
+
+    @responses.activate
+    def test_new_version_already_exists(self, mock_logger, mock_safe_post_to_slack):
+        DataVersions('Clinvar', '2025-06-30').save()
+        responses.add(responses.GET, WEEKLY_XML_RELEASE, status=200, body=gzip.compress(WEEKLY_XML_RELEASE_DATA.encode()), stream=True)
+        call_command('reload_clinvar_all_variants')
+        mock_logger.assert_called_with('Clinvar ClickHouse tables already successfully updated to 2025-06-30, gracefully exiting.')
+
+    @responses.activate
+    def test_parse_variants_all_types(self, mock_logger, mock_safe_post_to_slack):
+        KeyLookupSnvIndel.objects.using('clickhouse_write').create(
+            variant_id='22-28695219-G-A',
+            key_id=12,
+        )
+        KeyLookupMito.objects.using('clickhouse_write').create(
+            variant_id='M-123-GA',
+            key_id=8,
+        )
```

Comment:
it would be better for you to create json fixture data for these tests and use that instead of creating all the models at test setup like this